### DISCUSSION TITLE
Log In With Google?

### DISCUSSION THREAD

**Discussion by SOULV1CE** [mrlt8/docker-wyze-bridge#1055] [Q&A] [answered]
Is it possible to enable login with Google? This the my current way of authenticating directly with Wyze as it is an option on their log in page. However if I try to use the same Google credentials manually it does not work.

https://auth.wyze.com/login
![image](https://github.com/mrlt8/docker-wyze-bridge/assets/23405670/afe9a4c1-b15c-48b2-9a2f-7c9155984de1)

**Comment by mrlt8** [comment, marked as answer]
Unfortunately, not right now. The oauth-based login seems to callback to the Wyze app so we'd need a way to intercept that to get the token. #1000


You'll need to link an email/password somewhere in your account so that you can login using the API.

**Comment by blistovmhz** [comment]
I also signed up with a google account via oauth. Creating an API key thus doesn't help.
So I created a new account not using google, and it doesn't work either. Feels like a Wyze issue. IS there really no way to use this without Wyze's cloud servers?
